src/server/logging/do_tags_match.js

```diff
@@ -0,0 +1,24 @@
+import { isArray } from 'lodash';
+
+export function doTagsMatch(event, expectedTags) {
+  if (!event || !isArray(event.tags)) {
+    return false;
+  }
+
+  if (event.tags.length !== expectedTags.length) {
+    return false;
+  }
+
+  const unmatchedEventTags = event.tags.slice(0);
+  const unmatchedExpectedTags = [];
```

Comment:
The tags seem to have a perceived order. Is there a reason for not requiring the order to match as well?